### PR TITLE
fix: publish to OSSHR repository

### PR DESCRIPTION
**Pull request by yshyn-iohk**
### Description: 
In this PR, the build file was fixed to add the metadata to the pom.xml files and publish these packages to the OSSHR repository.
Despite removing the `if` statement and filtering the project that should be published, the number of packages is the same. Refer to the [previous publication](https://central.sonatype.com/search?q=io.iohk.atala.prism.apollo) and [the new publication of the beta version](https://central.sonatype.com/search?q=org.hyperledger.identus.apollo)
 

### Checklist: 
- [x] My PR follows the [contribution guidelines](https://github.com/hyperledger/identus-apollo/blob/main/CONTRIBUTING.md) of this project
- [ ] My PR is free of third-party dependencies that don't comply with the [Allowlist](https://toc.hyperledger.org/governing-documents/allowed-third-party-license-policy.html#approved-licenses-for-allowlist)
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked the PR title to follow the [conventional commit specification](https://www.conventionalcommits.org/en/v1.0.0/)

### CODE REVIEW

**Review by patlo-iog — APPROVED** [review]

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -42,6 +42,7 @@ jobs:
 
       - name: "Install Java ${{ env.JAVA_VERSION }}"
         uses: actions/setup-java@v4
+        continue-on-error: true
```
Comment (FabioPinheiro) [inline .github/workflows/release.yml:45]: wait
why this?

Comment (yshyn-iohk) [reply .github/workflows/release.yml:45]: Thanks for asking.
For some reason, the release flow fails at the Post Install Task
For instance, here: https://github.com/hyperledger/identus-apollo/actions/runs/10279883067
The entire release fails even if the packages are published and the `chore` commit is pushed.

I could not find a solution for this now except to add `continue-on-error` to the `Post Install Java 17` task.

image

Comment (yshyn-iohk) [reply .github/workflows/release.yml:45]: https://github.com/hyperledger/identus-apollo/issues/189 for tracking

**Pull request merged**